recipes/zeromq/zeromq.inc

```diff
@@ -0,0 +1,21 @@
+inherit autotools pkgconfig c++ auto-package-utils auto-package-libs
+
+RECIPE_TYPES = "machine sdk"
```

Comment:
No, I just cannot imagine why/how zeromq would make sense in sdk context. May I ask what sdk:zeromq is used for?